halide.cmake

```diff
@@ -615,13 +615,17 @@ function(_halide_add_exec_generator_target EXEC_TARGET)
 
   add_custom_target(${EXEC_TARGET} DEPENDS ${args_OUTPUTS})
 
-  # As of CMake 3.x, add_custom_command() recognizes executable target names in its COMMAND.
```

Comment:
Unfortunately I could not get this to work with env variables at all. So I went back here to using $<TARGET_FILE:${args_GENERATOR_BINARY}> to execute the generator instead of directly referring to its target name only.